web/src/app/administration/components/workbasket-access-items/workbasket-access-items.component.spec.ts

```diff
@@ -15,19 +15,19 @@ import { ICONTYPES } from 'app/models/type';
 
 
 import { GeneralModalService } from 'app/services/general-modal/general-modal.service';
-import { SavingWorkbasketService } from 'app/administration/services/saving-workbaskets/saving-workbaskets.service';
+import { SavingWorkbasketService } from 'app/administration/services/saving-workbaskets.service';
 import { WorkbasketService } from 'app/shared/services/workbasket/workbasket.service';
 import { AlertService } from 'app/services/alert/alert.service';
 import { RequestInProgressService } from 'app/services/requestInProgress/request-in-progress.service';
 import { AccessIdsService } from 'app/shared/services/access-ids/access-ids.service';
 import { FormsValidatorService } from 'app/shared/services/forms/forms-validator.service';
 import { NgxsModule, Store } from '@ngxs/store';
 import { EngineConfigurationSelectors } from 'app/store/engine-configuration-store/engine-configuration.selectors';
-import { AccessItemsComponent } from './access-items.component';
+import { WorkbasketAccessItemsComponent } from './workbasket-access-items.component';
 
 describe('AccessItemsComponent', () => {
```

Comment:
We could change description to "WorkbasketAccessItemsComponent" since we renamed the component